launcher/electron/build.gradle

```diff
@@ -0,0 +1,146 @@
+/*
+ *  Copyright 2015 TWO SIGMA OPEN SOURCE, LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+task makeDist2(type: Exec, dependsOn: ':dev:build') {
+  def osName = System.properties['os.name']
+  def platform = 'windows'
+  if (osName.startsWith('Mac')) {
+    platform = 'mac'
+  }
+  if (osName.startsWith('Linux')) {
+    platform = 'ubuntu'
+  }
+
+  def zipName = 'dist'
+  if (platform == 'windows') {
+    workingDir '../../core'
+    commandLine 'cmd', '/c', '7za', 'a',
+                ('../launcher/electron/' + zipName + '.zip'), 'config', 'src/main/web', 'build/install/core', 'nginx', 'LICENSE',
+                'beaker.command.bat', '-xr!.~', '-xr!.gradle'
+  } else {
+    commandLine './makeDist', 'dist'
+  }
+}
+
+task clean(dependsOn: ['cleanApp', 'cleanDist']) {
+}
+
+task cleanApp(type: Exec) {
+  commandLine 'rm', '-rf', 'Beaker.app'
+}
+
+task cleanDist(type: Exec) {
+  commandLine 'rm', '-rf', 'dist'
+}
+
+task makeMacInstaller(type: Exec, dependsOn: 'makeMacElectron') {
+  commandLine './makeMacInstaller'
+}
+
+task makeMacElectron(dependsOn: ['clean', 'makeDist2', 'mCopyElectron', 'mCopyBeakerApp', 'mCopyDist', 'mCopyPList', 'mCopyIcons', 'mCopyJRE', 'mCopyTools']) {
+  println "Starting Electron build"
+}
+
+task mUnzipElectron(type: Exec) {
+  commandLine 'unzip', '-uo', 'electron-v0.28.1-darwin-x64.zip'
```

Comment:
Yep. I have a refactor of this gradle script on my TODO list. It is not particularly good-looking.